src/common/resolve.rs

```diff
@@ -210,7 +210,7 @@ pub(crate) fn resolve_path(command: &Path, path: &str) -> Option<PathBuf> {
     // replaced, or passed unchanged to the program that sudo executes.
 
     path.split(':')
```

Comment:
There is also `std::env::split_paths` which does the right thing depending on the OS, but given that we only support unixes that use `:` as path separator, I don't think it matters here.